### DISCUSSION TITLE
Update external templates to reference .NET 5 RC 2

### DISCUSSION THREAD

**Discussion by mikecasas** [oqtane/oqtane.framework#820] [Q&A]
@sbwalker Would you want me to update the references in the external templates?

**Comment by sbwalker** [comment]
The external templates have Nuget package references to Oqtane release packages which have dependencies on .NET Core 3.1. So if you update the module references to .NET 5 I  am not sure if it will work? You can try it.

**Reply by mikecasas** [reply]
That makes sense. I know there is still discussion on-going, but do plan on doing any intermediate Oqtane release, like a .NET 5 RC2? Or do you just plan on doing an Oqtane V2 release in Nov. 2020?

If the plan is one release, then I can wait until after to update the templates. Let me know, just trying to lighten the load.
